compiler/passes/checkParsed.cpp

```diff
@@ -66,6 +66,25 @@ checkParsed() {
               USR_FATAL_CONT(def->sym,
                              "Variable '%s' is not initialized or has no type",
                              def->sym->name);
+    }
+
+    //
+    // This test checks to see if query domains (e.g., '[?D]') are
+    // used in places other than formal argument type specifiers.
+    //
```

Comment:
Largely FYI and not a request to edit.  It is increasingly the convention to prefer
isFnSymbo(), which returns a bool, rather than toFnSymbol(), which returns NULL
or a FnSymbol* in this context.